src/bidiMapper/utils/mapperTabPage.ts

```diff
@@ -0,0 +1,82 @@
+/**
+ * Copyright 2022 Google LLC.
+ * Copyright (c) Microsoft Corporation.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { LogType } from '../../utils/log';
+
+export class MapperTabPage {
+  // HTML source code for the user-facing Mapper tab. Manually minified from `mapper.html`.
```

Comment:
Will we need to keep this up-to-date in the future, or was it an one-off operation?

Reply:
Removed the `mapper.html` file to keep a single source-of-truth.